### PR TITLE
Input bounding boxes

### PR DESCRIPTION
**Pull request by NateV**
Added a little formlet for manually adding boxes. When you click "add" in the formlet, the box is also added visually to the map.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +121,35 @@ <h3>Step 2: Select Bounding Boxes</h3>
                         <button class="btn btn-info" id="newBoxButton">New Box</button> &nbsp;&nbsp;&nbsp; <button class="btn btn-danger" id="cancel">Cancel</button>
                       </div>
                     </div>
+                  </div>
```
Comment (NateV) [inline index.html:124]: This part is my change

File: static/scripts/map.js [hunk 2]
```diff
@@ -116,6 +127,41 @@ function toggleBoxMarker(box) {
   }
 }
 
+function manuallyAddBox() {
```
Comment (NateV) [inline static/scripts/map.js:135]: This part is my change

**Comment by NateV** [comment]
I added comments where I made changes that weren't just auto style changes.

I guess in the future, I should separate the style auto-changes into their own prelim. commit.

**Comment by mhollander** [comment]
Having used this, I don't like how when you press "new box" a form appears below that seems to require you to type in a box name and lat/lon.  It isn't clear that this is an optional thing.  Is it possible that when you select new box, a button appears that says "manual input" which would then open up the form?  It took me a second to realize that I could just draw on the map, and I already know how this works.

**Comment by mhollander** [comment]
Or maybe there could be two buttons ("draw box" and "enter coordinates for box")

File: index.html [hunk 3]
```diff
@@ -127,25 +127,25 @@ <h3>Step 2: Select Bounding Boxes</h3>
...
-                            <div class='offset-1 col-2'>
                                 <input id='boxName' type='text'/>
                             </div>
                           </div>
                           <div class="row">
-                            <div class='col-1'> NE: (lat, lon)
-                              <input id='northEastLat' type='text'/> ,
-                              <input id='northEastLong' type='text'/>
-                            </div>
-                            <div class='col-1'> SE: (lat, lon)
```
Comment (mhollander) [inline index.html]: Should be SW, right?

File: index.html [hunk 4]
```diff
@@ -127,25 +127,25 @@ <h3>Step 2: Select Bounding Boxes</h3>
...
-                              <input id='southWestLat' type='text'/> ,
-                              <input id='southWestLon'/>
-                            </div>
+                            <div class='col-1'> NE: (lat, lon) </div>
+                            <input id='northEastLat' type='text'/> ,
+                            <input id='northEastLong' type='text'/>
+                          </div>
+                          <div class="row">
+                            <div class='col-1'> SE: (lat, lon) </div>
```
Comment (mhollander) [inline index.html]: SW (same as above)

**Comment by NateV** [comment]
Improved w/ 713b65b?

**Review by mhollander — APPROVED** [review]
Everything looks good.

**Pull request merged**